downstream/modules/analytics/proc-add-roles-to-group.adoc

```diff
@@ -14,7 +14,7 @@ Adding roles to an existing group will grant new permissions to existing users i
 . Click image:images/cog.png[] > *Settings*.
 . From the sidebar, navigate to *User Access* > *Groups*.
```

Comment:
This should be rewritten to:

From the navigation bar, select menu:User Access[Groups].